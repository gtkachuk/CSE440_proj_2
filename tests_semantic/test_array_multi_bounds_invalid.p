program testArrayMultiBoundsInvalid;

class testArrayMultiBoundsInvalid

BEGIN
   
VAR
   bad	: ARRAY[4..9] OF ARRAY[28..38] OF integer;

FUNCTION testArrayMultiBoundsInvalid;
BEGIN
   PRINT bad[5,28];
   bad[5,27] := 67;
   PRINT bad[3,39]

END

END
.
