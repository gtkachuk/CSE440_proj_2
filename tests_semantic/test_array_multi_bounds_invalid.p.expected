ERROR:13:Array index 27 is out of the range (28,38)
ERROR:16:Array index 3 is out of the range (4,9)
ERROR:16:Array index 39 is out of the range (28,38)
Errors detected. Exiting.
